src/main/java/uk/gov/dvsa/mot/fixtures/mot/DocumentStepDefinitions.java

```diff
@@ -35,10 +35,28 @@ public DocumentStepDefinitions(WebDriverWrapper driverWrapper) {
         And("^I click \"([^\"]+)\" and check the PDF contains:$",
                 (String link, DataTable table) -> assertTrue(pdfContainsData(link, table)));
 
+        And("^I click the certificate link and check the MOT certificate PDF contains:$",
```

Comment:
Please add this new step definition to the docs

mot-automated-testsuite/docs/step-definitions.md

Reply:
Added.